frontend/src/types/schemas.ts

```diff
@@ -39,6 +39,10 @@ const integrationTypes = [
   "integrations.datadog.list_security_signals",
   "integrations.datadog.update_security_signal_state",
   "integrations.datadog.list_detection_rules",
+  "integrations.virustotal.get_file_report",
+  "integrations.virustotal.get_url_report",
+  "integrations.virustotal.get_domain_report",
+  "integrations.virustotal.get_ip_address_report"
 ] as const
```

Comment:
Need to add `virustotal` to  `integrationPlatforms` list on line 48/52.

https://github.com/TracecatHQ/tracecat/pull/79/files#diff-a1fdd3775730d4dc70c2d023e55cc9d3e5b50fa4354440f74ffe878835dc0d25L48